nautobot/ipam/tests/test_models.py

```diff
@@ -318,6 +318,10 @@ def test_get_utilization(self):
             # Create 32 IPAddresses within the Prefix
             [IPAddress(address=netaddr.IPNetwork("10.0.0.{}/24".format(i))) for i in range(1, 33)]
         )
+        # Create IPAddress objects for network and broadcast addresses
+        IPAddress.objects.bulk_create(
+            (IPAddress(address=netaddr.IPNetwork("10.0.0.0/32")), IPAddress(address=netaddr.IPNetwork("10.0.0.255/32")))
+        )
         self.assertEqual(prefix.get_utilization(), (32, 254))
```

Comment:
Looks like we're lacking any significant IPv6 testing in this file :( Would you mind adding a test function to do a similar test for IPv6 addresses (and assert that the 0 and all-ones addresses *are* counted towards IPv6 utilization)?

Reply:
Added tests for
- ipv4 pool prefix
- ipv4 /31 prefix
- ipv6 prefix